### ISSUE TITLE
Support for debian 10 (or even sid)

### ISSUE THREAD

**Issue by tsolar** [zakird/wkhtmltopdf_binary_gem#96]
I am using Debian sid, and the binary is not working out-of-the-box.

I saw https://github.com/zakird/wkhtmltopdf_binary_gem/pull/60 works only for Docker. I guess this is missing on the `bin/wkhtmltopdf` file

```ruby
os_based_on_debian = os.start_with?('debian_')
os = 'debian_10' if os_based_on_debian
```
Or something like that. I've monkey-patched the binary with it and it works.

Anyone does know if there is any better way to do it?

Thanks in advance!

**Comment by unixmonkey** [comment]
@tsolar That looks right. There are Debian 10 binaries, so a PR with that change would be appreciated!

**Comment by unixmonkey** [comment]
Thank you. This is now published in version 0.12.6.4.
